src/encoded/static/components/filegallery.js

```diff
@@ -220,19 +331,25 @@ export class FileTable extends React.Component {
                                 adminUser,
                             }}
                         />
-                        <SortTable
+                        {[analysisObjectKeys, otherKeys].map((keys) => keys.map((key) => <SortTable
```

Comment:
Very minor, but seems odd to see `<SortTable` way over on the right when it encapsulates everything to its left.

Reply:
I am not sure I understand this. The SortTable (line 223) is indeed on the same level as the code in line 334; they are siblings.

Reply:
Your line 334 has `<SortTable` after the arrow on the right side, but its props are way over on the left.

Reply:
I think it’d be less awkward to have:
```
{[analysisObjectKeys, otherKeys].map((keys) => keys.map((key) => (
    <SortTable
        prop1={prop1}
        prop2={props}
        ...
     >
```

Reply:
Done.  Thanks. Are you okay with the many commits thats part of this PR or do you prefer a rebase to just. get one commit to focus on?

Reply:
Many commits is fine, but whatever works for you.